pkgs/development/compilers/llvm/common/bolt/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv , llvm_meta
+, monorepoSrc , runCommand
+, cmake, libxml2, libllvm, libclang, version, python3
+, buildLlvmTools
+, additionalPatches ? []
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bolt";
+  inherit version;
+
+  # Blank llvm dir just so relative path works
+  src = runCommand "llvm-src-${version}" { } ''
```

Comment:
Please use the `src` derivation naming convention we use which is `"${pname}-src-${version}"`.